version.txt

```diff
@@ -4,9 +4,9 @@ COPYRIGHT_YEARS 2005-2018
 WEBSITE http://kodi.tv
 VERSION_MAJOR 18
 VERSION_MINOR 0
-VERSION_TAG RC5
-VERSION_CODE 17.99.952
-ADDON_API 17.9.952
+VERSION_TAG
+VERSION_CODE 18.00.000
```

Comment:
> error APPX0501: Validation error. error C00CE169: App manifest validation error: The app manifest must be valid as per schema: Line 10, Column 120, Reason: '18.00.000.0' violates pattern constraint of '(0|[1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])(\.(0|[1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])){3}'. The attribute 'Version' with value '18.00.000.0' failed to parse.

Reply:
This is going to be really annoying because it wants to get 18.0.0 however for android we should have 18.0.000 else it will totally screw up the version code we need.

Reply:
The zeros don't change anything for Android.
https://github.com/xbmc/xbmc/blob/2c44473e66a15b58ea66f71a49d5e024d0a85280/cmake/scripts/android/Install.cmake#L42-L47